pandas/core/indexing.py

```diff
@@ -1818,6 +1816,9 @@ def error():
             except:
                 error()
```

Comment:
In my last few commits this except clause hid the cause of the underlying exception. Cant it just be removed altogether and let exceptions raise as they occur onstead of goinc through error()?

Reply:
> In my last few commits this except clause hid the cause of the underlying exception.

Well, in principle this is its exact goal ;-) Can you provide an example? I guess we should just specify the exceptions it catches.

Reply:
> Cant it just be removed altogether and let exceptions raise as they occur onstead of goinc through error()?

@topper-123 well, [maybe you're right](https://github.com/pandas-dev/pandas/pull/21594)!